assets/js/components/dashboard-sharing/DashboardSharingSettings/Module.js

```diff
@@ -142,6 +144,7 @@ export default function Module( { moduleSlug, moduleName, ownerUsername } ) {
 					'googlesitekit-dashboard-sharing-settings__row--disabled': isLocked,
 				}
 			) }
+			ref={ moduleRef }
```

Comment:
Sorry, why is this ref being assigned to two different components? That seems off. 🤔

Reply:
@tofumatt I'm passing the `moduleRef` onto the child `UserRoleSelect` component which is wrapped with `forwardRef`, i.e. it is assigned to the `div`, but just being passed to `UserRoleSelect` to use with `useKeyCodesInside`. Does it look good?

Reply:
Ohhh, I see. Yup, that makes sense, thanks! 👍🏻